### ISSUE TITLE
A way to transition a file group from downloader so something less heavy on the session

### ISSUE THREAD

**Issue by alidan** [hydrusnetwork/hydrus#972]
I Find downloaders VERY useful for grouping up certain types of files to cull at a time rather than just dealing with random chunk.

however I think most of the info from a file when its downloaded is useless after the fact, so my question is would it be possible to have the file group transition to something less heavy by user input (select the group and transition them, i'm thinking about other people who may find that info useful so an auto may not be good for them) 

If something like this could be applied to both gallery searches and watchers (I think with simple downloaders you can do something like this manually, highlighting all the files and opening them in a new page, at least it seems to be able to take a 4 million weight group and moves them to 40~ thousand) I think it would be the last downloader upgrade i would need.

**Comment by Zweibach** [comment]
Sounds like this is what you want, not sure if it was added after the feature request or not but it's in now at least. If so, close the issue. If not, expand on it further.
![X6qjkwgMJb](https://user-images.githubusercontent.com/10709426/137583455-1b44f155-009d-4b9e-8281-7d8d6823b024.png)
> I think with simple downloaders you can do something like this manually, highlighting all the files and opening them in a new page

Decently sure you've always been able to do this no matter the page.

**Comment by alidan** [comment]
yes, for downloaders and I did something similar to this before I posted this, I do find this useful for downloaders, however watchers and gallery, currently a few thousand watchers on the current session totalling in at a weight of about 8.5 million and if they were transferred to something lighter, like just a file window, they likely wouldn't even hit 300k weight. but opening a few thousand pages is non viable, and moving all the files to massive pages would remove the compartmentalized nature of them. 

here I'll download a for instance set

ok, I downloaded a few threads for ic for the example
![client_2021-10-16_19-08-23](https://user-images.githubusercontent.com/2681758/137605076-0f788b10-a16e-4754-bacd-fa7f2466748e.png)

these threads are all still active, so you will have to imagine they are completed threads, 
as it stands right now, they have a collective file count of 124 but they have a weight of 2660

I personally find the distinctions between the threads very useful and would want to keep them, however keeping in this case 8 pages (in my watchers list about 2500) is cumbersome, or in the case of some boards the topics come up many times, so I may want to deal with the same topics in one larger list rather then smaller pagesets... this is why im wondering if there would be a way to implement a way to keep the file set without all the baggage that adds that much weight. 

GRANTED this is assuming those weight numbers are correct and the page with some files open actually is only 124 weight apposed to 2660 (as the example above)
I mean if the weight is wrong, then this idea kind of needs to be reworked, but if it actually is that low of weight then going forward within the current watcher/gallery downloader could work.

**Comment by Zweibach** [comment]
Clear the URL logs, they're the ones that provide most of the weight. A way to clear the URLs for a completed watcher/gallery query from right-clicking the item in the menu would make it a bit more convenient I guess. As for grouping stuff, we have tags and I'm not sure why people keep ignoring those for organising downloads for the purpose of dealing with a batch of stuff later like you seem to want to. There's little reason to keep all those pages open as far as I can tell when you can assign some tag and search up later.

**Comment by alidan** [comment]
the main problems with tags is there is no convenient way to tag the images for parsing. I mean I could tag them according to thread subject, but then going though images, especially from an image board where they may or not be on topic now creates a substantial amount work when you are doing them one at a time, even galleries the search term depending on how well tagged the search was would give false positives. 

I mean if you are dealing with a small group of images and want to put it off for later, ok, tags or ratings may work for putting it off, my hydrus has 6,159,805 images in it as of now, as far as I can tell there is no clean way to parse images with tags thats anything reasonably fast. 

my system currently revolves around setting up rating gates and star ratings. 
parsing images based on 'do I like it or not' puts an extra star rating on it, so I can filter rated and not rated, along with a 0-10 star rating, the same image can pass through the rating multiple times, and get a higher star rating for passing the visual 'do I like it' check multiple times, once it hits 10 it can be archived as I oked it several times though a few different moods. 
rating gates would be something like 'real/drawn' or 'safe/suggestive/explicit' the first layer of these has 2 gates due to how easy it is to set these gates up, any incoming image would get these two and it would mean it can move on to the next level of processing, which would be another place a gate comes into effect, essentially sorting images rapidly into categories for a mass tag addition because ratings as far as I can tell are easier to manipulate with a hotkey then tags are. 

due to the massive amount of images I have I put a hold on rating gate parsing in favor of 'do I like it' parsing, however this method relies on downloaders and galleries separating images into categories.

with that said, how do you clear url logs, and can I do it to a single watcher/gallery to see if it has the intended effect?
